examples/chip-tool/commands/clusters/ClusterCommand.h

```diff
@@ -107,6 +112,36 @@ class ClusterCommand : public ModelCommand, public chip::app::CommandSender::Cal
         return CHIP_NO_ERROR;
     }
 
+    template <class T>
+    CHIP_ERROR SendGroupCommand(chip::GroupId groupId, chip::FabricIndex fabricIndex, chip::NodeId senderNodeId,
+                                chip::ClusterId clusterId, chip::CommandId commandId, const T & value)
+    {
+        chip::app::CommandPathParams commandPath = { 0 /* endpoint */, groupId, clusterId, commandId,
+                                                     (chip::app::CommandPathFlags::kGroupIdValid) };
+
+        chip::Messaging::ExchangeManager * exchangeManager = chip::app::InteractionModelEngine::GetInstance()->GetExchangeManager();
+
+        ChipLogDetail(chipTool, "Sending command to Group %u, on Fabric %x, for cluster %u with commandId %u", groupId, fabricIndex,
+                      clusterId, commandId);
+
+        auto commandSender = chip::Platform::MakeUnique<chip::app::CommandSender>(this, exchangeManager, false);
+        VerifyOrReturnError(commandSender != nullptr, CHIP_ERROR_NO_MEMORY);
+        ReturnErrorOnFailure(commandSender->AddRequestDataNoTimedCheck(commandPath, value, chip::NullOptional));
```

Comment:
Should be passing `mTimedInteractionTimeoutMs` so we can use this to test (invalid) timed group commands.

Reply:
done